qcodes/dataset/threading.py

```diff
@@ -11,18 +11,18 @@
 from collections import defaultdict
 from functools import partial
 from types import TracebackType
-from typing import TYPE_CHECKING, Callable, List, Sequence, TypeVar, Union
+from typing import TYPE_CHECKING, Callable, List, Sequence, Tuple, TypeVar, Union
 
 from typing_extensions import Protocol
 
 from qcodes.utils import RespondingThread
 
 if TYPE_CHECKING:
-    from qcodes.dataset.measurements import res_type
+    from qcodes.dataset.measurements import values_type
```

Comment:
its actually from data_set_protocol, right?
```suggestion
    from qcodes.dataset.data_set_protocol import values_type
```

Reply:
Yes looking forward to no implicit reexport